### DISCUSSION TITLE
Enums

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [dicebear/dicebear#228] [Q&A] [answered]
Would be nice to use enums or to get all Values of a specific style.

```js
{
...
clothes: Clothes.blazer
...
}
```

or

```js
let clothValues = Object.keys(Clothes);
```

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @NilsBaumgartner1994 

Each avatar style comes with a JSON schema. From this JSON schema you can read the options.

```js
import * as style from '@dicebear/avataaars';

let clothValues = style.schema.properties.clothes.items.enum;

/**
[
    "blazer",
    "blazerAndShirt",
    "blazerAndSweater",
    "sweater",
    "collarAndSweater",
    "shirt",
    "graphicShirt",
    "shirtCrewNeck",
    "shirtScoopNeck",
    "shirtVNeck",
    "hoodie",
    "overall"
]
*/
```

See also:
- https://github.com/dicebear/dicebear/discussions/204

If you are looking for autocompletion, you could use Typescript or Visual Studio Code.
